test/signalflow/request_manager.js

```diff
@@ -7,13 +7,15 @@ var expect = require('chai').expect;
 describe('The SignalFlow Request Manager', function () {
   //TODO : re-enable this test once we figure out why it causes a hang
   var client = sflowclient('AUTHTOKEN');
-  it('should initialize a request manager with three methods, execute, stop, authenticate', function (done) {
+  it('should initialize a request manager with four methods, execute, stop, authenticate, registerLiveTail and stopLiveTail', function (done) {
```

Comment:
So... is it four methods like the text says, five methods like the list says, or 6 methods like the test expects? Come on :)

Reply:
fixed :)